smart_contracts/xgov_registry/contract.py

```diff
@@ -149,6 +149,9 @@ def __init__(self) -> None:
         self.pending_proposals = GlobalState(UInt64(), key=cfg.GS_KEY_PENDING_PROPOSALS)
 
         self.request_id = GlobalState(UInt64(), key=cfg.GS_KEY_REQUEST_ID)
+        self.request_unsubscribe_id = GlobalState(
+            UInt64(), key=cfg.GS_KEY_REQUEST_UNSUBSCRIBE_ID
+        )
```

Comment:
The new variable initialization in the `__init__` might not be executed in the scope and Application Update. We should confirm if the Global State initialization is performed both on App Create _and_ App Update.